criu/sk-unix.c

```diff
@@ -1170,6 +1170,7 @@ static int revert_unix_sk_cwd(struct unix_sk_info *ui, int *prev_cwd_fd, int *ro
 	if (*ns_fd >= 0 && restore_ns(*ns_fd, &mnt_ns_desc))
 		ret = -1;
 	if (*root_fd >= 0) {
+		/* coverity[chroot_call] */
```

Comment:
I don't understand what warnings  mean in the commit message. Could you add a more detailed explanation and add a link to the coverity issue? Thanks!